frontend/npm-shrinkwrap.json

```diff
@@ -1,19303 +1,9 @@
 {
   "name": "openproject-frontend",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
@oliverguenther what (apart from `npm install`) do I have to do to create something with `lockfileVersion` 2?

Reply:
I have no clue why it would remove the shrinkwrap. Please just `git checkout origin/dev -- frontend/npm-shrinkwrap.json` as you have made no changes to the package.json. Usually, the shrinkwrap should auto-update once added dependencies.